### PR TITLE
multi-arch: create build scripts for multi-arch

### PR DESCRIPTION
**Pull request by huoqifeng**
Signed-off-by: huoqifeng <huoqif@cn.ibm.com>

### CODE REVIEW

**Comment by huoqifeng** [comment]
@rootfs @SamYuan1990 may you help review the scripts to build multi-arch images?

File: hack/build-images.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+function create_image_manifest_base () {
+	docker manifest create \
+		${registry}/${image}:latest \
+		${registry}/${image}:latest-amd64 \
+		${registry}/${image}:latest-s390x \
+		${registry}/${image}:latest-arm64
+
+	docker manifest push ${registry}/${image}:latest
```
Comment (rootfs) [inline hack/build-images.sh:100]: on systems without docker, podman is the way to go

**Comment by rootfs** [comment]
@huoqifeng on systems without docker installed, we use [buildah and podman](https://podman.io/blogs/2021/10/11/multiarch.html), can you see if both can be supported? thanks

**Comment by huoqifeng** [comment]
`buildah` vs `docker`: https://danmanners.com/posts/2022-01-buildah-multi-arch/

**Comment by huoqifeng** [comment]
I'd suggest rename build-images.sh to `build-images-docker.sh` and provide a new scripts for `buildah` as this tends to be used by developer.

**Review by rootfs — APPROVED** [review]

**Comment by rootfs** [comment]
sounds good, looking forward to the dev scripts.

**Comment by marceloamaral** [comment]
We should update the makefile at some point to use this script

**Pull request merged**